libs/blocks/global-navigation/utilities/menu/menu.css

```diff
@@ -197,4 +201,51 @@
   .feds-promo--dark a:hover {
     text-decoration: underline;
   }
+
+
+  .feds-links-strip-wrapper {
+    width: 100%;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: 100%;
+    border-top: solid 1px #f3f3f3;
+  }
+
+  .feds-links-strip {
+    width: 100%;
+    padding: 20px 0;
+    max-width: var(--feds-maxWidth-nav);
+    display: flex;
+    justify-content: flex-start;
+    align-items: center;
+  }
+
+  .feds-links-strip > *:not(:last-child)::after {
+    content: '|';
+    margin: 0;
+  }
+
+  .feds-links-strip-item {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+  }
+
+  .feds-popup .feds-navLink:hover {
+    background-color: transparent;
```

Comment:
This is likely an error, we definitely want background colours for dropdown links, it's an accessibility (and therefore legal) requirement. You probably meant to cancel the background colour just for navLink elements from the linksStrip?

Reply:
Yes, it's an error. I wanted to cancel the background only for the elements inside the linksStrip